lib/elixir/lib/exception.ex

```diff
@@ -261,6 +261,11 @@ defmodule Exception do
       {:%{}, meta, [__struct__: Range, first: first, last: last, step: step]} ->
         {:"..//", meta, [first, last, step]}
 
+      # TODO: Remove me on v2.0
+      {:%{}, meta, [__struct__: Range, first: first, last: last]} ->
+        step = if first <= last, do: 1, else: -1
+        {:"..//", meta, [first, last, step]}
```

Comment:
We don't need this clause because then it will be printed as the struct when formatting arguments in the error message, which is fine.

Reply:
In this case Macro.prewalk invokes callback with `{:%{}, meta, [__struct__: Range, first: first, last: last, step: step]}` so it's unnecessary